lib/console.js

```diff
@@ -33,7 +35,8 @@ function Console(stdout, stderr) {
 }
 
 Console.prototype.log = function() {
-  this._stdout.write(util.format.apply(this, arguments) + '\n');
+  this._stdout.write(util.format.apply(this, arguments).replace(/^/gm,
+          Array(this._group * 2 + 1).join(' ')) + '\n');
```

Comment:
If you want performance, define `this._group` as string.
in `group` function do `this._group += '  '`
in this function do:
```js
var data = util.format.apply(this, arguments);
if (this._group.length) data = data.replace(/^/gm, this._group);
this._stdout.write(data + '\n');
```